src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,55 @@
+import cn from 'classnames';
+import React from 'react';
+import { FilterType } from '../../types/FilterType';
+
+type Props = {
+  changeFilterStatus: (status: FilterType) => void;
+  status: FilterType;
+};
+
+export const Filter: React.FC<Props> = ({
+  changeFilterStatus, status,
+}) => {
+  const filterAll = () => changeFilterStatus(FilterType.All);
+  const filterActive = () => changeFilterStatus(FilterType.Active);
+  const filterCompleted = () => changeFilterStatus(FilterType.Completed);
```

Comment:
do you need three separated functions? You can do it with one, just pass needed filter inside